src/query/functions/src/scalars/map.rs

```diff
@@ -227,4 +231,22 @@ pub fn register(registry: &mut FunctionRegistry) {
         |_, _| FunctionDomain::Full,
         |map, _| map.len() as u64,
     );
+
+    registry.register_2_arg_core::<EmptyMapType, StringType, BooleanType, _, _>(
```

Comment:
The type of the second argument should be generic type. The key of the map also supports other types, such as int.

Reply:
Indeed, I overlooked this type.